### ISSUE TITLE
SickChill Fails to Build Due to Removal of codecov Package From PyPi

### ISSUE THREAD

**Issue by ecliptik** [SickChill/sickchill#8416]
SickChill is no longer building from source due to the removal of `codecov` from PyPi [1].

https://about.codecov.io/blog/message-regarding-the-pypi-package/

Version: 2023.1.2
OS: Linux
Browser: NA
What you did: Build SickChill from Source
What happened: Fails with missing codecov package
What you expected: Successful Build
Logs:
```
$HOME/.cargo/env && poetry install --no-root --no-interaction --no-ansi                                                                   
 Creating virtualenv sickchill in /app/sickchill/.venv                                                                                                          
 Updating dependencies                                                                                                                                          
 Resolving dependencies...                                                                                                                                      
                                                                                                                                                            
 Because sickchill depends on codecov (^2.1.12) which doesn't match any versions, version solving failed. 
```

**Comment by miigotu** [comment]
Where is that log from?

**Comment by ecliptik** [comment]
It's from a custom `Dockerfile` based off one that was in the repo prior to https://github.com/SickChill/sickchill/commit/d8c180a832e72e77791036fcbbca40764ff5c28a that originally had the `RUN . $HOME/.cargo/env && poetry install --no-root --no-interaction --no-ansi` line.

I did build it using the latest `Dockerfile` in `master` to test and it did not give the error. However I noticed that that `Dockerfile` has a malformed command on line 71 https://github.com/SickChill/sickchill/blob/master/Dockerfile#L71 that causes the build to fail.

I'll update my customer Dockerfile to more closely match the one in `master` and can close this out.
